app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="header">
+    <div class="act">
+      <%= link_to 'Add Recipe', new_recipe_path %>
+      <%= link_to "New recipe_food", new_recipe_food_path %>
+    </div>
+ 
+
+  <% @recipes.each do |recipe| %>
+    <div class="recipe">
+      <div class="action">
+      <h3><%= recipe.name %></h3>
+        <%= link_to 'REMOVE', recipe, method: :delete, class: 'remove-btn' %>
+      </div>
+      <div class="description">
+        <p><%= recipe.description %></p>
+      </div>
```

Comment:
- [ ] Kindly make the name of a recipe to be a link so that it is easy for users to go back and be able to generate shopping lists. Currently, users are typing the URL for a specific recipe to access it (show page).

![links](https://github.com/ikoote1/Recipe-App/assets/53124548/7831c698-63eb-4bec-8d20-3e2488f4d12d)